tests/gnrc_tcp/Makefile

```diff
@@ -0,0 +1,47 @@
+include ../Makefile.tests_common
+
+# Basic Configuration
+BOARD ?= native
+PORT ?= tap0
+
+# Shorten default TCP timeouts to speedup testing
+MSL_US ?= 1000000
+TIMEOUT_US ?= 3000000
+
+# Mark Boards with insufficient memory
+BOARD_INSUFFICIENT_MEMORY :=
+
+CFLAGS += -DSHELL_NO_ECHO
+CFLAGS += -DGNRC_TCP_MSL=$(MSL_US)
+CFLAGS += -DGNRC_TCP_CONNECTION_TIMEOUT_DURATION=$(TIMEOUT_US)
+
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_ipv6_default
+USEMODULE += gnrc_tcp
+USEMODULE += shell
+USEMODULE += shell_commands
+
+# Setup ethos if BOARD is not "native".
+ifeq (,$(filter native,$(BOARD)))
+ETHOS_TTY ?= /dev/ttyACM0
```

Comment:
This works only for boards that pick this port name (e.g. not for `iotlab-m3` and not on Mac OSX. Just use `PORT` as provided by the `board`'s config or the user via the environment.